adhocteam/Head-Start-TTADP
# Add AU-2 specified audit log messages

**Description of change**

Add AUDIT label messages for easier filtering in server logs and adds various interesting messages to that log.

**How to test**

* Run server locally and check log for AUDIT messages around login/logout and admin routes.


**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/261

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
